advertising/email-00-registration.md

```diff
@@ -0,0 +1,30 @@
+Title: Our Networks: registration open + vend at the Yami-ichi ⚑
```

Comment:
I would save this file as `.txt`, not `.md`

Reply:
Did you see my comment on the previous PR? 

We've been consistently .md until that one .txt, I'm not seeing a strong reason for one over the other as whoever sends the message has a lot of final control on formatting? Raw md is still _unformatted_ but reads nicer when viewed on gh

Reply:
Sorry I missed the one in the previous PR. When it's an .md two lines get folded into one, that's why I prefer keeping as plain text, but I am okay with keeping things consistent as is.

Reply:
fair point. Lets merge for now and refactor?